fieldtofile/fieldtofile-service.ts

```diff
@@ -1,5 +1,12 @@
 import * as fs from "fs";
 import * as PH from "processhub-sdk";
+import { getBackendUrl } from "processhub-sdk/lib/config";
+
+async function getAttachedFile(environment: PH.ServiceTask.IServiceTaskEnvironment, relativePath: string): Promise<Buffer> {
+  const pathParts = relativePath.split("/");
+  const filePath = decodeURIComponent(relativePath.replace(pathParts[0] + "/", "").replace(pathParts[1] + "/", ""));
+  return await environment.fileStore.getFileBuffer(pathParts[0], pathParts[1], filePath);
```

Comment:
This feels dangerous to me as it modifies the path based on the number of slashes here. Also, I'm not 100% sure if it does not break with https://github.com/roXtra/processhub-sdk/pull/150.
Probably it would be better to add a method to `config` in SDK that returns the wanted part as we could probably also need this in other locations later.
Please at least comment what path comes in here as `relativePath` and what path is extracted with this operation and doublecheck if this still works with the mentioned changes in SDK.